dbms/src/AggregateFunctions/AggregateFunctionSum.cpp

```diff
@@ -18,9 +18,12 @@
 #include <AggregateFunctions/Helpers.h>
 #include <fmt/core.h>
 
-
 namespace DB
 {
+
+extern const String count_second_stage = NameCountSecondStage::name;
+extern const String sum_on_partial_result_stage = NameSumOnPartialResult::name;
```

Comment:
why there is `_stage` in `sum_on_partial_result`?

Reply:
> why there is `_stage` in `sum_on_partial_result`?

I will delete it